Doc/tools/templates/download.html

```diff
@@ -40,7 +40,7 @@ <h1>Download Python {{ release }} Documentation</h1>
 
 <p>These archives contain all the content in the documentation.</p>
 
-<p>HTML Help (<tt>.chm</tt>) files are made available in the "Windows" section
+<p>HTML Help (.chm) files are made available in the "Windows" section
```

Comment:
Shouldn't we should use something like `<span style="'font-family:monospace' ">.chm</span>`?

Reply:
If we're going to replace it with anything, I think this would be appropriate:

```suggestion
<p>HTML Help (<code>.chm</code>) files are made available in the "Windows" section
```

But that's more of a formatting argument. If needed, I'm okay with just removing it since the `<tt>` tag is obsolete and could be removed at any point in time; which is more of an immediate concern. I also don't think it's especially impactful for an extension name to be formatted in a manner that's separate from the rest of the text.